wp-content/themes/goonj-crm/functions.php

```diff
@@ -191,7 +191,30 @@ function goonj_handle_user_identification_form() {
 		}
 
 		// If we are here, then it means the user exists as an inducted volunteer.
-		wp_redirect(get_home_url() . "/collection-camp-form/#?source_contact_id=" . $contact['id'] );
+		// Fetch the most recent collection camp activity based on the creation date
+		$collectionCampResult = civicrm_api3('Activity', 'get', [
+			'sequential' => 1,
+			'contact_id' => $contact['id'],
+			'activity_type_id' => ['IN' => [61]], // ID for "Collection Camp Intent"
+			'status_id' => ['IN' => [10]], // Status ID for "Under Authorization"
```

Comment:
Why are you using `IN` operator when there is a single value. Why not directly `=`